### PR TITLE
Updated notebook and markdown

### PR DESCRIPTION
**Pull request by Vismayak**

### CODE REVIEW

**Comment by navarroc** [comment]
We're there any updates to the notebook? It's minor, but I think you can clear all the outputs cells and commit that or revert to the original unchanged notebook. The only notebooks we've included results are the testbed notebooks.

**Comment by navarroc** [comment]
Can you update the description section of the markdown file too? We should change "The outputs of this analysis are a CSV file with maximum damage state from each hazard and the overall maximum damage." to 

"The outputs of this analysis are a CSV file with maximum damage state from each hazard and the overall maximum damage and a CSV with the combined damage probabilities from the three hazards.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Updated pyincore-data example with new census util [#183](https://github.com/IN-CORE/incore-docs/issues/183)
 - Remove incorrect characterization of population dislocation dataset [185](https://github.com/IN-CORE/incore-docs/issues/185)
-
+- Update combined damage analysis documentation [#194](https://github.com/IN-CORE/incore-docs/issues/194)
```
Comment (longshuicy) [inline CHANGELOG.md:12]: can you add the empty line back after this entry?

File: notebooks/combined_wind_wave_surge_building_dmg.ipynb [hunk 2]
```diff
@@ -2,9 +2,13 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (longshuicy) [inline notebooks/combined_wind_wave_surge_building_dmg.ipynb:5]: I see weird execution count everywhere... haha how did you remove the output cell???
image


Could you try Kernel "restart and clear output"?
image

**Review by longshuicy — CHANGES_REQUESTED** [review]

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**